lms/static/js/fixtures/instructor_dashboard/data_download.html

```diff
@@ -1,9 +1,225 @@
+
+        <li class="nav-item ">
+            <button type="button" class="btn-link reports  active-section" data-section="reports">Reports</button>
+        </li>
+        <li class="nav-item">
+            <button type="button" class="btn-link problem-report" data-section="problem">Problem Report</button>
+        </li>
+        <li class="nav-item">
+            <button type="button" class="btn-link certificates" data-section="certificate">Certificates</button>
+        </li>
+        <li class="nav-item">
+            <button type="button" class="btn-link grading" data-section="grading">Grading</button>
+        </li>
+
+    </ul>
+    <section id="reports" class="idash-section tab-data" aria-labelledby="header-reports">
+        <h6 class="mb-15" id="header-reports">
+            <strong>${_("NOTE")}: </strong>
+            Please select the report type and then click Download Report button
+        </h6>
+        <div class="mb-15">
+
+            <div class="mb-5">
+                <select class="report-type selector">
+                    <option value="gradingConfiguration"
+                            data-endpoint="${ section_data['get_grading_config_url'] }">
+                        Grading Configuration
+                    </option>
+                    <option value="listAnonymizeStudentIDs" data-endpoint="${ section_data['get_anon_ids_url'] }"
+                            class="${'is-disabled' if disable_buttons else ''}"
+                            aria-disabled="${'true' if disable_buttons else 'false'}">Anonymized Student IDs
+                    </option>
+                    <option value="profileInformation"
+                            data-endpoint="${ section_data['get_students_features_url'] }"
+                            data-csv="true">Profile Information
+                    </option>
+                    <option value="learnerWhoCanEnroll"
+                            data-endpoint="${ section_data['get_students_who_may_enroll_url'] }" data-csv="true">
+                        Learner
+                        who can enroll
+                    </option>
+                    <option value="listEnrolledPeople"
+                            data-endpoint="${ section_data['get_students_features_url'] }">
+                        List enrolled students profile information
+                    </option>
+                    <option value="proctoredExamResults"
+                            data-endpoint="${ section_data['list_proctored_results_url'] }">Proctored exam results
+                    </option>
+                    <option value="surveyResultReport"
+                            data-endpoint="${ section_data['course_survey_results_url'] }">
+                        Survey Result report
+                    </option>
+                    <option value="ORADataReport" data-graderelated="true"
+                            data-endpoint="${ section_data['export_ora2_data_url'] }">ORA Data
```

Comment:
is this variable useful? `data-endpoint="${ section_data['export_ora2_data_url'] }"` here?  ++ others in this file?